### PR TITLE
Clean up header monitor menu

### PR DESCRIPTION
**Pull request by moshthepitt**
- It seems that IRS performance report menu item was removed, so added it back
- Cleaned up enableMonitorDropDown which referenced ENABLE_IRS_PERFORMANCE_REPORT twice
- Cleaned up Header component --> mainly removed dead links
- Group IRS related menu items for consistency
- Cleaned up mocked env vars
- Closes https://github.com/onaio/reveal-frontend/issues/1385

menu monitor

### CODE REVIEW

**Review by KipSigei — APPROVED** [review]

**Review by p-netm — APPROVED** [review]

**Review by kelvin-muchiri — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,8 +42,8 @@ jobs:
           REACT_APP_IRS_PLAN_COUNTRIES: 'NA,TH'
           REACT_APP_EXPRESS_OAUTH_GET_STATE_URL: 'http://localhost:3000/oauth/state'
           REACT_APP_EXPRESS_OAUTH_LOGOUT_URL: 'http://localhost:3000/logout'
-          REACT_APP_BACKEND_ACTIVE: true
-          REACT_APP_ENABLE_MDA_POINT: true
+          REACT_APP_BACKEND_ACTIVE: 'true'
+          REACT_APP_ENABLE_MDA_POINT: 'true'
```
Comment (moshthepitt) [inline .circleci/config.yml:46]: Having these without the quote raises lint warnings

File: .circleci/config.yml [hunk 2]
```diff
@@ -16,9 +16,9 @@ jobs:
           REACT_APP_THEME_COLOR: '000000'
           REACT_APP_ENABLE_IRS: 'true'
           REACT_APP_ENABLE_FI: 'true'
-          REACT_APP_ENABLE_USERS: 'true'
+          REACT_APP_ENABLE_USERS: 'false'
           REACT_APP_ENABLE_ASSIGN: 'true'
-          REACT_APP_ENABLE_ABOUT: 'true'
+          REACT_APP_ENABLE_ABOUT: 'false'
```
Comment (moshthepitt) [inline .circleci/config.yml:21]: Disabling these as they are never used

File: src/components/page/Header/index.tsx [hunk 3]
```diff
@@ -318,13 +329,6 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
                         </NavLink>
                       </DropdownItem>
                     )}
-                    {ENABLE_USERS && (
```
Comment (moshthepitt) [inline src/components/page/Header/index.tsx:321]: Removing this placeholder nav link

File: src/components/page/Header/index.tsx [hunk 4]
```diff
@@ -396,14 +400,6 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
                   </DropdownMenu>
                 </UncontrolledDropdown>
               )}
-
-              {ENABLE_ABOUT && (
```
Comment (moshthepitt) [inline src/components/page/Header/index.tsx:400]: Removing this placeholder nav link

File: src/configs/__mocks__/env.ts [hunk 5]
```diff
@@ -213,10 +190,23 @@ export const PLAN_TYPES_WITH_MULTI_JURISDICTIONS = [
 
 export const SHOW_TEAM_ASSIGN_ON_OPERATIONAL_AREAS_ONLY = false;
 
+// Menu items
```
Comment (moshthepitt) [inline src/configs/__mocks__/env.ts:193]: Consolidating similar variables for clarity and ease of maintenance

**Review by KipSigei — APPROVED** [review]

**Review by p-netm — APPROVED** [review]

**Pull request merged**